src/platform/input/game-pads.js

```diff
@@ -68,62 +109,619 @@ const MAPS = {
         ],
 
         axes: [
-            // Analogue Sticks
+            // Analog Sticks
             'PAD_L_STICK_X',
             'PAD_L_STICK_Y',
             'PAD_R_STICK_X',
             'PAD_R_STICK_Y'
         ]
+    },
+
+    DEFAULT_XR: {
+        buttons: [
+            // Back buttons
+            'XRPAD_TRIGGER',
+            'XRPAD_SQUEEZE',
+
+            // Axes buttons
+            'XRPAD_TOUCHPAD_BUTTON',
+            'XRPAD_STICK_BUTTON',
+
+            // Face buttons
+            'XRPAD_A',
+            'XRPAD_B'
+        ],
+
+        axes: [
+            // Analog Sticks
+            'XRPAD_TOUCHPAD_X',
+            'XRPAD_TOUCHPAD_Y',
+            'XRPAD_STICK_X',
+            'XRPAD_STICK_Y'
+        ]
     }
 };
 
 const PRODUCT_CODES = {
     'Product: 0268': 'PS3'
 };
 
+let deadZone = 0.25;
+const previous = [];
+
+/**
+ * A GamePadButton stores information about a button from the Gamepad API.
+ */
+class GamePadButton {
+    /**
+     * Create a new GamePadButton instance.
+     *
+     * @param {GamepadButton} button - The original Gamepad API gamepad button.
+     * @hideconstructor
+     */
+    constructor(button) {
+        /**
+         * Whether this button was pressed on last frame.
+         *
+         * @type {boolean}
+         * @ignore
+         */
+        this._previouslyPressed = button.pressed;
```

Comment:
This looks like `this.pressed`, while...

Reply:
It's changed, thanks!